src/Middleware/src/Headstart.Common/Assets/EmailTemplates/CriticalSupport.html

```diff
@@ -0,0 +1,408 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html data-editor-version="2" class="sg-campaigns" xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+    <title>Order Submitted</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1" />
+    <!--[if !mso]><!-->
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <!--<![endif]-->
+    <!--[if (gte mso 9)|(IE)]>
```

Comment:
I usually count commented code as tech debt, is there a reason we need to keep these in? im noticing there's a lot of commented out code in these templates so maybe there's a reason for it.

Reply:
These comments are used for browser compatibility. They test which browser the user has and enable the sections within the if blocks. [More info here](https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/compatibility/ms537512(v=vs.85)?redirectedfrom=MSDN#downlevel-hidden-conditional-comments)